build.gradle

```diff
@@ -1,58 +1,169 @@
+tasks.withType(JavaCompile) {
+    options.compilerArgs << "-Xlint:deprecation" << "-Xlint:unchecked"
+    options.encoding = "UTF-8"
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(8)
+    }
+
+    withJavadocJar()
+    withSourcesJar()
+}
 
 dependencies {
-    compile  "org.embulk:embulk-core:0.7.0"
-    provided "org.embulk:embulk-core:0.7.0"
+    compileOnly "org.embulk:embulk-spi:0.11"
+    implementation "org.embulk:embulk-util-config:0.5.0"
+    implementation "org.embulk:embulk-util-file:0.2.0"
     // compile "YOUR_JAR_DEPENDENCY_GROUP:YOUR_JAR_DEPENDENCY_MODULE:YOUR_JAR_DEPENDENCY_VERSION"
-    testCompile "junit:junit:4.+"
-    testCompile 'org.embulk:embulk-core:0.7.+:tests'
-}
-
-task classpath(type: Copy, dependsOn: ["jar"]) {
-    doFirst { file("classpath").deleteDir() }
-    from (configurations.runtime - configurations.provided + files(jar.archivePath))
-    into "classpath"
-}
-clean { delete 'classpath' }
-
-task gem(type: JRubyExec, dependsOn: ["build", "gemspec", "classpath"]) {
-    jrubyArgs "-rrubygems/gem_runner", "-eGem::GemRunner.new.run(ARGV)", "build"
-    script "build/gemspec"
-    doLast { ant.move(file: "${project.name}-${project.version}.gem", todir: "pkg") }
-}
-
-task gemspec << { file("build/gemspec").write($/
-Gem::Specification.new do |spec|
-  spec.name          = "${project.name}"
-  spec.version       = "${project.version}"
-  spec.authors       = ["Sadayuki Furuhashi"]
-  spec.summary       = %[Command file output plugin for Embulk]
-  spec.description   = %[Executes commands and writes files to STDIN.]
-  spec.email         = ["frsyuki@gmail.com"]
-  spec.licenses      = ["Apache 2.0"]
-  spec.homepage      = "https://github.com/embulk/embulk-output-command"
-
-  spec.files         = `git ls-files`.split("\n") + Dir["classpath/*.jar"]
-  spec.test_files    = spec.files.grep(%r"^(test|spec)/")
-  spec.require_paths = ["lib"]
-
-  #spec.add_dependency 'YOUR_GEM_DEPENDENCY', ['~> YOUR_GEM_DEPENDENCY_VERSION']
-  spec.add_development_dependency 'bundler', ['~> 1.0']
-  spec.add_development_dependency 'rake', ['>= 10.0']
-end
-/$)
+
+    testImplementation "junit:junit:4.13.2"
+    testImplementation "org.embulk:embulk-core:0.11.3"
+    testImplementation "org.embulk:embulk-deps:0.11.3"
+    testImplementation "org.embulk:embulk-junit4:0.11.3"
+}
+
+embulkPlugin {
+    mainClass = "org.embulk.output.CommandFileOutputPlugin"
+    category = "output"
+    type = "command"
+}
+
+gem {
+    authors = [ "Sadayuki Furuhashi" ]
+    email = [ "frsyuki@gmail.com" ]
+    // "description" of the gem is copied from "description" of your Gradle project.
+    summary = "Command file output plugin for Embulk"
+    homepage = "https://github.com/embulk/embulk-output-command"
+    licenses = [ "Apache-2.0" ]
+
+    from rootProject.file("LICENSE")
+}
+
+gemPush {
+    host = "https://rubygems.org"
+}
+
+jar {
+    metaInf {
+        from rootProject.file("LICENSE")
+    }
+}
+
```

Comment:
`javadocJar`?

Reply:
Thanks fixed in 7f3bcedc83318a5886268b5a2ccf7ac3723d3d96